fs/expose/dokan/__init__.py

```diff
@@ -5,19 +5,18 @@
 Expose an FS object to the native filesystem via Dokan.
 
 This module provides the necessary interfaces to mount an FS object into
-the local filesystem using Dokan on win32::
+the local filesystem using Dokan on win32:
```

Comment:
Looks like this double-colon _is_ actually needed? See http://docs.pyfilesystem.org/en/latest/expose/dokan.html